app/assets/stylesheets/mobile/markdown_editor.scss

```diff
@@ -0,0 +1,22 @@
+.md-editor {
+  border: 1px solid $light-grey;
+  border-radius: $btn-border-radius-base;
+  overflow: hidden;
+
+  &.active { border-color: $text-grey; }
+
+  &::after,
+  &::before {
+    clear: both;
+    content: ' ';
+    display: table;
+  }
+
+  textarea {
+    border: 0;
+    border-radius: 0;
```

Comment:
The desktop markdown editor CSS already removes the border, so one can remove these two lines.